src/bin/tools/addImportAndSwitchToKCPage.ts

```diff
@@ -0,0 +1,199 @@
+import ts from "typescript";
+import * as fs from "fs";
+// import * as prettier from "prettier";
+
+export const addImportAndSwitchToKCPage = async (
+    kcPagePath: string,
+    caseName: string,
+    reactComponentName: string,
+    isFormField: boolean
+): Promise<void> => {
+    let content = fs.readFileSync(kcPagePath).toString("utf-8");
+    const sourceFile = ts.createSourceFile(
+        "temp.tsx",
+        content,
+        ts.ScriptTarget.ESNext,
+        true,
+        ts.ScriptKind.TSX
+    );
+    const result = ts.transform(sourceFile, [
+        typescriptTransformer(caseName, reactComponentName, isFormField)
+    ]);
+    const printer = ts.createPrinter();
+    const transformedSourceFile = result.transformed[0] as ts.SourceFile;
+    content = printer.printFile(transformedSourceFile);
+
+    //TODO Unfortunately this will cause build error after compilation so we can't prettify kcPage before printing for now
+    //Prettify ts file before returning
+    // let formatedContent = await prettier.format(content, {
```

Comment:
Just a note that unfortunately, I didn’t manage to prettify the file before saving because of a build error with the Prettier API. However, this shouldn’t be a problem since the starter project already includes Prettier. If anyone wants to edit the file, they can do it manually.